src/tools/schema2base.ts

```diff
@@ -35,7 +35,7 @@ export interface ClassGenerator {
 export class NodeAndGenerator {
   node: SchemaNode;
   generator: ClassGenerator;
-  hash: string;
+  hash: string | null;
```

Comment:
Given that in the case of unconstrained type variables, we want to generate the empty schema, how about I make the schema value of SchemaNode and this hash non-null?
@piotrswigon